src/main/java/org/terasology/potions/system/DrinkPotionAuthoritySystem.java

```diff
@@ -217,6 +217,9 @@ public void onPotionWithoutGenomeConsumed(DrinkPotionEvent event, EntityRef ref)
                 case PotionCommonEffects.EXPLOSIVE:
                     herbEffect = new ExplosiveEffect();
                     break;
+				case PotionCommonEffects.NO_VISIBILITY:
+                    herbEffect = new NoVisibilityEffect();
```

Comment:
You have the class now, but the DrinkPotionAuthoritySystem needs to import it before it'll be able to find it. Again something like IntelliJ should outright prompt you "Hey want to import this class from X?" :-) 

You also have tab characters again on the case statement line